Util/Tokenization/src/main/java/org/tribuo/util/tokens/impl/SplitCharactersTokenizer.java

```diff
@@ -154,28 +160,31 @@ private static boolean isCharacter(char c, char[] chars) {
 
     /**
      * Returns a copy of the split characters.
+     * 
      * @return A copy of the split characters.
      */
     @Deprecated
     public char[] getSplitCharacters() {
-        return Arrays.copyOf(splitCharacters,splitCharacters.length);
+        return Arrays.copyOf(splitCharacters, splitCharacters.length);
     }
 
     /**
      * Returns a copy of the split characters except inside digits.
+     * 
      * @return A copy of the split characters.
      */
     @Deprecated
     public char[] getSplitXDigitsCharacters() {
-        return Arrays.copyOf(splitXDigitsCharacters,splitXDigitsCharacters.length);
+        return Arrays.copyOf(splitXDigitsCharacters, splitXDigitsCharacters.length);
     }
 
     @Override
     public SplitCharactersTokenizer clone() {
-        char[] sc = splitCharacters == null ? null : Arrays.copyOf(splitCharacters,splitCharacters.length);
-        char[] sxc = splitXDigitsCharacters == null ? null : Arrays.copyOf(splitXDigitsCharacters,splitXDigitsCharacters.length);
+        char[] sc = splitCharacters == null ? null : Arrays.copyOf(splitCharacters, splitCharacters.length);
```

Comment:
Given these arrays aren't mutated, we could probably get away with them all sharing a reference to the same one to save some memory, but it's probably not worthwhile.

Reply:
it simplifies the clone method to a single line if we make the suggested change.  So, I don't see the harm in it and it cleans the code up a bit.